public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -130,7 +129,7 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
 
   public countPercentage(countValue: number) {
     if (!countValue || countValue <= 0) { return 0; }
-    let percent: number = countValue / this.survey.countValue ;
+    let percent: number = countValue / this.survey.participantCount ;
     percent = parseFloat(percent.toFixed(2));
```

Comment:
Can you change this to 4 and see what happens ? Because we end up with a lot of 0% in the display for answers not answered a lot. Better to see 0.01 ... Just simply change that to 2 to a 4